### PR TITLE
Try to draft code of conduct

### PR DESCRIPTION
**Pull request by PsiACE**
- Portions derived from [the Docker Code of Conduct](https://github.com/docker/code-of-conduct/) and [the Angular Contributor Code of Conduct](https://github.com/angular/code-of-conduct).
- I’m not quite sure about the scope of `Apache-2.0`. Since this is text material, I may prefer `CC-BY-4.0`.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Portions derived from [the Docker Code of Conduct](https://github.com/docker/code-of-conduct/) and [the Angular Contributor Code of Conduct](https://github.com/angular/code-of-conduct).
+
+## License
+
+This work is licensed under a Creative Commons Attribution 4.0 Unported License
+
+For attribution requirements:
+
+"@Casbin Code of Conduct" © 2020 Casbin, used under a Creative Commons Attribution Unported license: http://creativecommons.org/licenses/by/4.0/"
```
Comment (zhmushan) [inline CODE_OF_CONDUCT.md:48]:
```suggestion
"@Casbin Code of Conduct" © 2020 Casbin, used under a Creative Commons Attribution Unported license: http://creativecommons.org/licenses/by/4.0/
```

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+
+Portions derived from [the Docker Code of Conduct](https://github.com/docker/code-of-conduct/) and [the Angular Contributor Code of Conduct](https://github.com/angular/code-of-conduct).
+
+## License
+
+This work is licensed under a Creative Commons Attribution 4.0 Unported License
+
+For attribution requirements:
+
+"@Casbin Code of Conduct" © 2020 Casbin, used under a Creative Commons Attribution Unported license: http://creativecommons.org/licenses/by/4.0/"
```
Comment (zhmushan) [inline README.md:28]:
```suggestion
"@Casbin Code of Conduct" © 2020 Casbin, used under a Creative Commons Attribution Unported license: http://creativecommons.org/licenses/by/4.0/
```

**Comment by hsluoyz** [comment]
lgtm

**Comment by hsluoyz** [comment]
@PsiACE we can change the license to CC4 if wanted.

**Comment by PsiACE** [comment]
> @PsiACE we can change the license to CC4 if wanted.

I do think CC-BY-4.0 is more suitable for code of conduct.

**Pull request merged**